### PR TITLE
Add a shell endpoint that provides fees for the transaction

### PR DESCRIPTION
**Pull request by MichaelKim20**
I added a shell endpoint that provides fees for the transaction
The endpoint will be updated after implementing the ability to find appropriate fees.

Relates to #274

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -114,6 +115,7 @@ class Stoa extends WebService
         this.app.get("/transaction/pending/:hash", this.getTransactionPending.bind(this));
         this.app.get("/utxo/:address", this.getUTXO.bind(this));
         this.app.get("/transaction/status/:hash", this.getTransactionStatus.bind(this));
+        this.app.get("/transaction/fees/:size", this.getTransactionFees.bind(this));
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
        this.app.get("/transaction/fees/:tx_size", this.getTransactionFees.bind(this));
```
It's easier to understand.

File: src/Stoa.ts [hunk 2]
```diff
@@ -366,6 +368,51 @@ class Stoa extends WebService
...
+     * Called when a request is received through the `/transaction/fees/:size` handler
+     *
+     * Returns transaction fees by the transaction size.
+     */
+    private getTransactionFees (req: express.Request, res: express.Response)
+    {
+        // TODO This is still incomplete.
+        //  We will proceed with statistics to find appropriate fees and then upgrade.
+
+        let size: string = req.params.size.toString();
```
Comment (TrustHenry) [inline src/Stoa.ts]: How about inputting the size as a `number`?

File: src/Types.ts [hunk 3]
```diff
@@ -318,3 +318,29 @@ export class ConvertTypes
...
+
+/**
+ * Define the interface of the fee of the transaction
+ */
+export interface ITransactionFee
+{
+    /**
+     * The size of the transaction
+     */
+    size: number;
```
Comment (TrustHenry) [inline src/Types.ts]:
```suggestion
    tx_size: number;
```

**Comment by TrustHenry** [comment]
We can get the average transaction fee from the past blocks, not the shell function.

**Comment by MichaelKim20** [comment]
> We can get the average transaction fee from the past blocks, not the shell function.

Yes, that's right, I'll proceed with that next time.

File: src/Stoa.ts [hunk 4]
```diff
@@ -366,6 +368,51 @@ class Stoa extends WebService
...
+        // TODO This is still incomplete.
+        //  We will proceed with statistics to find appropriate fees and then upgrade.
+
+        let size: string = req.params.tx_size.toString();
+
+        logger.http(`GET /transaction/fees/${size}}`);
+
+        if (!Utils.isPositiveInteger(size))
+        {
+            res.status(400).send(`Invalid value for parameter 'size': ${size}`);
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
            res.status(400).send(`Invalid value for parameter 'tx_size': ${size}`);
```

File: src/Stoa.ts (hunk 2 above)
Comment (MichaelKim20) [reply src/Stoa.ts]: In the following row, check to see if it is a positive number. 
It is only possible when it is a string

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**